playground/infrastructure/cloudbuild/examples_cd.sh

```diff
@@ -0,0 +1,161 @@
+
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+   export "$KEY"="$VALUE"
+done
+
+# This block sets default values for several environment variables to run CD script.
+export ORIGIN=${ORIGIN-"PG_EXAMPLES"}
+export SUBDIRS=${SUBDIRS-"./learning/katas ./examples ./sdks"}
+export BEAM_ROOT_DIR=${BEAM_ROOT_DIR-"/workspace/beam"}
+export PROJECT_ID=${PROJECT_ID}
+export SDK_CONFIG=${SDK_CONFIG-"$BEAM_ROOT_DIR/playground/sdks.yaml"}
+export BEAM_EXAMPLE_CATEGORIES=${BEAM_EXAMPLE_CATEGORIES-"$BEAM_ROOT_DIR/playground/categories.yaml"}
+export BEAM_USE_WEBGRPC=${BEAM_USE_WEBGRPC-"yes"}
+export BEAM_CONCURRENCY=${BEAM_CONCURRENCY-2}
+export SDKS=${SDKS-"java python go"}
+export COMMIT=${COMMIT-"HEAD"}
+export DNS_NAME=${DNS_NAME}
+if [[ -z "${DNS_NAME}" ]]; then
+  echo "DNS_NAME is empty or not set. Exiting"
+  exit 1
+fi
+export NAMESPACE=${NAMESPACE}
+if [[ -z "${NAMESPACE}" ]]; then
+  echo "Datastore NAMESPACE is empty or not set. Exiting"
+  exit 1
+fi
+export SOURCE_BRANCH=${SOURCE_BRANCH}
+if [[ -z "${SOURCE_BRANCH}" ]]; then
+  echo "PR Source Branch is empty or not set. Exiting"
+  exit 1
+fi
+export ALLOWLIST=${ALLOWLIST-"learning/katas sdks/ examples/"}
+export DIFF_BASE=${DIFF_BASE-"origin/master"}
+
+# This function logs the given message to a file and outputs it to the console.
+function LogOutput ()
+{
+    echo "$(date --utc '+%D %T') $1" >> $LOG_PATH
+    # CDLOG keyword to simplify search over the global log
+    echo "CDLOG $(date --utc '+%D %T') $1"
+}
+
+LogOutput "Input variables:
+ ORIGIN=$ORIGIN
+ STEP=${STEP}
+ SUBDIRS=$SUBDIRS
+ BEAM_ROOT_DIR=$BEAM_ROOT_DIR
+ PROJECT_ID=$PROJECT_ID
+ SDK_CONFIG=$SDK_CONFIG
+ BEAM_EXAMPLE_CATEGORIES=$BEAM_EXAMPLE_CATEGORIES
+ BEAM_CONCURRENCY=$BEAM_CONCURRENCY
+ SDKS=$SDKS
+ COMMIT=$COMMIT
+ DNS_NAME=$DNS_NAME
+ BEAM_USE_WEBGRPC=$BEAM_USE_WEBGRPC
+ NAMESPACE=$NAMESPACE
+ SOURCE_BRANCH=$SOURCE_BRANCH"
+
+# Script starts in a clean environment in Cloud Build. Set minimal required environment variables
+if [ -z "$PATH" ]; then
+    export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin/:/usr/bin:/sbin:/bin"
+fi
+if [ -z "$HOME" ]; then
+    export HOME="/builder/home"
+fi
+
+export STEP=CD
+
+LogOutput "Installing python and dependencies."
+# Install Python 3 and dependencies
+set -e  # Exit immediately if any command fails
+apt update > /dev/null 2>&1
+export DEBIAN_FRONTEND=noninteractive
+apt install -y apt-transport-https ca-certificates software-properties-common curl unzip apt-utils > /dev/null 2>&1
+add-apt-repository -y ppa:deadsnakes/ppa > /dev/null 2>&1 && apt update > /dev/null 2>&1
+apt install -y python3.8 python3.8-distutils python3-pip > /dev/null 2>&1
+apt install -y --reinstall python3.8-distutils > /dev/null 2>&1
+pip install --upgrade google-api-python-client > /dev/null 2>&1
+python3.8 -m pip install pip --upgrade > /dev/null 2>&1
+ln -s /usr/bin/python3.8 /usr/bin/python > /dev/null 2>&1
+apt install -y python3.8-venv > /dev/null 2>&1
```

Comment:
what's the point in installing `python3.8-venv` if we are not using it after and installing pip packages globally anyway?